cert/cert.go

```diff
@@ -36,11 +37,29 @@ type CA struct {
 	AuthKey     string `json:"auth_key" yaml:"auth_key"`
 	AuthKeyFile string `json:"auth_key_file" yaml:"auth_key_file"`
 	File        *File  `json:"file,omitempty" yaml:"file,omitempty"`
+	RootCACert  string `json:"root_ca,omitempty" yaml:"root_ca,omitempty"`
 	pem         []byte
 }
 
 func (ca *CA) getRemoteCert() ([]byte, error) {
-	remote := client.NewServer(ca.Remote)
+	var tlsConfig tls.Config
+	if len(ca.RootCACert) > 0 {
```

Comment:
For consistency with other checks in this file (eg, `AuthKeyFile`), we might want to compare against an empty string, rather than a length check.

Reply:
Thanks @terinjokes.

Fixed in 519f8db36eca7e983b7a790fb0f24f94a01ca38e .. Will squash if you approve. Any other comments to the functionality in general? It does make sense to implement support for self-signed certs on cfssl-apiserver this way right?